modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionServiceImpl.java

```diff
@@ -93,6 +93,25 @@ public ListTypeDefinition getListTypeDefinition(long listTypeDefinitionId)
 			listTypeDefinitionId);
 	}
 
+	@Override
+	public ListTypeDefinition getListTypeDefinitionByExternalReferenceCode(
+			String externalReferenceCode, long companyId)
+		throws PortalException {
+
+		ListTypeDefinition listTypeDefinition =
+			listTypeDefinitionLocalService.
+				fetchListTypeDefinitionByExternalReferenceCode(
```

Comment:
Can you follow the same pattern of method `ObjectDefinitionServiceImpl#getObjectDefinitionByExternalReferenceCode`?
I understood that Liferay's pattern consists in **get**XXX (will raise an exception if the object doesn't exists) ou **fetch**XXX (will return null if the object doesn't exists). So if you change **fetch**ListTypeDefinitionByExternalReferenceCode by **get**ListTypeDefinitionByExternalReferenceCode , it will no longer be necessary to check if listTypeDefinition is not null.

Reply:
Sure!
Thanks for the tip!